sys/include/net/nanocoap.h

```diff
@@ -182,13 +181,24 @@ extern "C" {
  * @name    Content-Format option codes
  * @{
  */
-#define COAP_FORMAT_TEXT         (0)
-#define COAP_FORMAT_LINK        (40)
-#define COAP_FORMAT_OCTET       (42)
-#define COAP_FORMAT_JSON        (50)
-#define COAP_FORMAT_CBOR        (60)
-/** @brief   nanocoap-specific value to indicate no format specified. */
-#define COAP_FORMAT_NONE     (65535)
+#define COAP_FORMAT_TEXT            (0)
+#define COAP_FORMAT_LINK           (40)
+#define COAP_FORMAT_OCTET          (42)
+#define COAP_FORMAT_JSON           (50)
+#define COAP_FORMAT_CBOR           (60)
+/**
+ * @brief    nanocoap-internal value to indicate no format specified.
+ *
+ * Not intended for use in a transmitted packet.
+ */
+#define COAP_FORMAT_NONE        (65535)
```

Comment:
minor thing: we could improve readability slightly by using `UINT16_MAX` (needs `#include <limits.h>`) here and below.

Reply:
Good idea. Actually, UINT16_MAX is in <stdint.h>. I was surprised to learn this, too. ;-)